packages/lit-html/src/test/helpers_test.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {html, render} from '../lit-html.js';
+import {assert} from '@esm-bundle/chai';
+import {stripExpressionComments} from './test-utils/strip-markers.js';
+
+import {join, map, range, when} from '../helpers.js';
+
+suite('template helpers', () => {
+  let container: HTMLDivElement;
+
+  const assertRender = (value: unknown, expected: string) => {
+    render(value, container);
+    return assert.equal(stripExpressionComments(container.innerHTML), expected);
+  };
+
+  setup(() => {
+    container = document.createElement('div');
+  });
+
+  suite('when', () => {
+    test('true case, without false case', () => {
+      assertRender(
+        when(true, () => html`X`),
+        'X'
+      );
+    });
```

Comment:
Add a test that `` when(false, () => html`X`) `` returns `undefined`?

Reply:
(Sorry for nits)

I think this new `assertRender` should be in its own test function?

Also some of the descriptions might be a little unclear. E.g. there's "true case, with false case" and "false case", which are the same except for the first parameter. So I guess it's unclear to me if "case" is referring to the first parameter or the second two parameters.

Suggestions:

```
true with false case
false with false case
true without false case
false without false case
```

Or:

```
when(true, T)
when(false, T)
when(true, T, F)
when(false, T, F)
```

Reply:
fixed